crates/wasi-nn/Cargo.toml

```diff
@@ -17,7 +17,8 @@ anyhow = "1.0"
 wiggle = { path = "../wiggle", version = "=0.38.0" }
 
 # These dependencies are necessary for the wasi-nn implementation:
-openvino = { version = "0.3.3", features = ["runtime-linking"] }
+openvino = { version = "0.4.0", features = ["runtime-linking"] }
+tensorflow = { git = "https://github.com/tensorflow/rust.git", features = ["tensorflow_runtime_linking"]}
```

Comment:
Could both be made optional so you could include just one if necessary rather than forcing both?

Reply:
The entire wasi-nn module can be turned on and off; @brianjjones has worked on https://github.com/tensorflow/rust/pull/354 to make sure this code path should work regardless of whether TF libraries are present on the system.

My concern here is that, until the `tensorflow` crate releases a new version with that PR's changes, we should fix this dependency to a specific commit to avoid any upstream issues.